bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -134,7 +134,23 @@ case class LastStepConverter(semester: Semester) extends SemesterConverter {
  * This converter will upgrade to 2018A
  */
 case object SemesterConverter2017BTo2018A extends SemesterConverter {
-  override val transformers = Nil
+  val jLoFilter = "J-lo (1.122 um)"
+  val yFilter = "Y (1.020 um)"
+
+  val removeF2YJloFilters: TransformFunction = {
+    case p @ <flamingos2>{ns @ _*}</flamingos2> if (ns \ "filter").map(_.text).exists(f => f === jLoFilter || f === yFilter) =>
+      object yjLoFilterTransformer extends BasicTransformer {
+        override def transform(n: xml.Node): xml.NodeSeq = n match {
+          case <name>{name}</name>                                   => <name>{name.text.replaceAll(jLoFilter, "").replaceAll(yFilter, "")}</name>
+          case <filter>{filter}</filter> if filter.text == jLoFilter => xml.NodeSeq.Empty
+          case <filter>{filter}</filter> if filter.text == yFilter   => xml.NodeSeq.Empty
```

Comment:
What happens if the proposal is left without any filters?  Do you lose the rest of the blueprint information?